netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -749,10 +752,21 @@ _net_ebpf_sock_addr_clean_up_security_descriptor()
     }
 }
 
-static void
+static ebpf_result_t
 _net_ebpf_sock_addr_initialize_globals()
 {
+    ebpf_result_t ebpf_result = EBPF_SUCCESS;
+    ebpf_epoch_state_t epoch_state = {0};
+    const ebpf_hash_table_creation_options_t options = {
+        .key_size = EBPF_OFFSET_OF(net_ebpf_extension_connection_context_t, timestamp),
+        .value_size = sizeof(net_ebpf_extension_connection_context_t*)};
+
     InitializeListHead(&_net_ebpf_ext_connect_context_list);
+    ebpf_epoch_enter(&epoch_state);
+    ebpf_result = ebpf_hash_table_create(&_net_ebpf_ext_connect_context_hash_table, &options);
```

Comment:
Thanks! I did not know that this was an option. After looking more closely at the code, I see that we can pass in the allocate and free options when creating the hash table. I will try that out and see if there are any issues there. 

(I'd also like to avoid pulling in epoch if not necessary).